python/hidet/option.py

```diff
@@ -263,6 +263,12 @@ def register_hidet_options():
         default_value=True,
         description='Whether to enable imperative execution when op arguments allows',
     )
+    register_option(
+        name='store_dispatch_table',
+        type_hint='bool',
+        default_value=False,
+        description="Whether to save and load the CompiledGraph's dispatch table",
+    )
```

Comment:
I prefer to put the flag as a parameter to `save_compiled_graph` instead taking it as an option. Otherwise, our options will explode soon. Usually, only use `option` to place the option that will be accessed in multiple places across the whole project, or the ones that are very important and need to be exposed to users.